ignite/templates/app/files/app/app.go.plush

```diff
@@ -317,9 +312,6 @@ func New(
 	return app
 }
 
-// Name returns the name of the App
-func (app *App) Name() string { return app.BaseApp.Name() }
```

Comment:
I know removing this breaks scaffolding, but I am gladly removing it, as `App` implements `BaseApp`, so redefining it isn't necessary.